server/emailTemplates/verification.js

```diff
@@ -0,0 +1,40 @@
+import Mailgen from 'mailgen';
+import { tokenGenerator, getFullUrl, getBaseUrl } from '<helpers>/utils';
+
+const appName = process.env.APP_NAME;
```

Comment:
I appreciate the initiative of using an environment variable to keep this dynamic

but since app_Name is something that would not change based on the environment or has any security implications, or would not readily change.

I think you could use Uppercase for appName to differentiate it as a constant that would not change
i.e APP_NAme

I think it could be stored as a constant in place of an environment variable.

Reply:
Ok. Noted